pymonntorch/NetworkCore/Network.py

```diff
@@ -146,6 +152,10 @@ def __repr__(self):
                 result += str(sg) + "\r\n"
             used_tags.append(tags)
 
+        for struc in self.Structures:
+            if struc not in self.NeuronGroups and struc not in self.SynapseGroups:
+                result += str(struc) + "\r\n"
```

Comment:
is "carriage return" needed 😅 ?

Reply:
Apparently not, Old convention, gonna fix it in another PR.